Formula/b/bold.rb

```diff
@@ -0,0 +1,48 @@
+class Bold < Formula
+  desc "Drop-in replacement for Apple system linker ld"
+  homepage "https://github.com/kubkon/bold"
+  url "https://github.com/kubkon/bold/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "2496f04e47c9d5e17ef273b26519abf429b5c3e3df6d264f2941735088253ec0"
+  license "MIT"
+
+  depends_on "zig" => :build
+  depends_on :macos
```

Comment:
Seems like this should build on Linux too. Why make it macOS-only?

Reply:
I thought this is only for apple system linker replacement?

Reply:
It is, but that doesn't mean you can't run this on Linux. You could be cross-compiling, for example.

Reply:
yeah, let try it :)